bootstrap-conda

```diff
@@ -106,58 +106,18 @@ echo >&2 $0:$LINENO: generate conda environment files
         for pkg in $SAGELIB_SYSTEM_PACKAGES; do
             echo "  - $pkg"
         done
-    ) > src/environment-template.yml
-
-    (
-        cat environment-template.yml
-        echo "  # optional packages"
-        for pkg in $OPTIONAL_SYSTEM_PACKAGES; do
-            echo "  - $pkg"
-        done
-    ) > environment-optional-template.yml
+    ) > environment-template.yml
```

Comment:
@tobiasdiez This breaks the build.

https://unix.stackexchange.com/questions/110490/why-does-the-command-shuf-file-file-leave-an-empty-file-but-similar-commands


On the left hand side you read from `environment-template.yml`, but on the right hand side you `>` into that same file. On my machine the file is erased before it's read.

Fortunately the fix is simple: since both are actually the same file, you can just combine the two blocks into one.

Reply:
This is fixed in #37447